File: Javascript/Reverse_Words_in_a_String_III.js
// leet code challenge 'Reverse Words in a String III'
// problem link https://leetcode.com/problems/reverse-words-in-a-string-iii/description/

/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function (s) {
  let arr = s.split(" ");
  for (let i = 0; i < arr.length; i++) {
    arr[i] = arr[i].split("").reverse().join("");
  }
  return arr.join(" ");
};

console.log(reverseWords("Let's take LeetCode contest"));

File: C#/Reverse_Words_in_a_String_III.cs
// leet code challenge 'Reverse Words in a String III'
// problem link https://leetcode.com/problems/reverse-words-in-a-string-iii/description/

public class Solution {
    public string ReverseWords(string s) {
        string[] arr = s.Split(" ");
        for (int i = 0; i < arr.Length; i++)
        {
           char[] x = arr[i].ToCharArray();
            Array.Reverse(x);
            arr[i] = new string(x);
        }
        return string.Join(" ",arr);
    }
}